src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        HashMap<String, Integer> reportInfo = new HashMap<>(); //create values
```

Comment:
1) Comments in code are considered a very bad practice. Code itself is a text, code should be self-descriptive by method and variable names. You need to think twice or even better thrice before writing a comment in code
2) Please refer to the "common mistakes" file and don't use hashMap 
https://mate-academy.github.io/jv-program-common-mistakes/java-core/exceptions/salary-info#dont-use-two-dimension-arrays-and-hashmapor-any-other-map